cluster/gce/configure-vm.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
Oi .. as someone who looks at a ton of startupscript.logs, this is really spammy. :/ I worked for a while to make the boot script more concise (until the salt execution, but that's the last step), so 99% of the noise is now a bug.